R/take-snapshot.R

```diff
@@ -22,9 +22,12 @@ take_snapshot <- function(){
 
   # one level more for humans
   # since they use subfolders
-  cran_human <- c("DS", "UL", "SH", "KH")
+  cran_human <- c("BA", "DS", "KH", "KL", "SH", "UL", "VW")
```

Comment:
how could we keep this up-to-date more easily?

Reply:
Good call! 
It's done in [`b059c1d` (#78)](https://github.com/r-hub/cransays/pull/78/commits/b059c1daab1658f88fb5480c0507304efc688d8b).

Reply:
:ok_hand: